### PR TITLE
Weekly Changelog 2023-07-24 to 2023-07-31

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -40,6 +40,10 @@ Apply different messages at different effect intensity levels
 Player character now can get sick with common cold or flu only after contacts with NPCs or ferals
 Generate vehicle prototypes from in-game vehicles
 In-game Armor sprite change
+Cockroaches flee light, hide under furniture, eat corpses, and breed faster when fed
+Add some dialog function
+Auto Eat/Drink zones now prioritize less-fresh items
+Player activity related event
```
Comment (dseguin) [inline data/changelog.txt:46]:
```suggestion
Cockroaches flee light, hide under furniture, eat corpses, and breed faster when fed
```

File: data/changelog.txt [hunk 2]
```diff
@@ -72,6 +76,23 @@ Adds new skateboard vehicle
...
+Remove not used `welding component kit` item
+Updated the description for Chelsea Boots to use the correct usage of 'donning' vs. the incorrect 'dawning'
+Make bone broth better by extracting more calcium using acified water
+adding nylon quivers with MOLLE capabilities
+house roadmap: update house_01 in-construction variant to use modern code
+Miscellaneous recipe category fixes/additions
+More zombie harvest snippets
+Solar Farm
+Two new civilians: Parent and hospitalized varieties
+Add Electric Masonry Saw
```
Comment (dseguin) [inline data/changelog.txt:95]:
```suggestion
Adds a mutable stream to the mapgen
```

File: data/changelog.txt [hunk 3]
```diff
@@ -122,6 +144,14 @@ Add the Mind Over Matter mod to the CDDA repository
...
 [Sky Island] Mainline Sky Island mod
 [Magiclysm] Add a spell-using feral human to Magiclysm
+Blood Purge removes mutagens, you can heal followers' diseases
+Change the Utility Molding spell to allow selecting the starting tool
+Fix monster power messages
+Allow telekinetics to push very heavy objects and lift cars
+Add u_message to the extremely important note
+Add psionic civilians to the initial Cataclysm chaos waves
+Convert recipe powers into spell powers, add new powers
+Add three new powers
```
Comment (dseguin) [inline data/changelog.txt:154]:
```suggestion
```

File: data/changelog.txt [hunk 4]
```diff
@@ -147,6 +177,19 @@ Lycra is less protective
...
+Adjusts ranges of SMGs and pistol caliber carbines
+More storage for belts
+Reduce energy requirements for making powdered milk
+Workshop Elves will fight back in the off chance they are pushed into a corner
+More monsters fight back if cornered
+Adjusts Bovine Bulk description to portray it as a defensive mutation
+Make bone inedible
+Price changes for various items as per the documentation
+Balance steel flak helmets
+higher dodge skill lowers the stamina cost to dodge
```
Comment (dseguin) [inline data/changelog.txt:192]:
```suggestion
More monsters fight back if cornered
Higher dodge skill lowers the stamina cost to dodge
```

File: data/changelog.txt [hunk 5]
```diff
@@ -206,6 +249,30 @@ Don't get randomly sick anymore
...
+Correct some wrong IDs for Cody's armor projects
+Stop aphids from eating their own honeydew
+Fix grainy glyphs in blended font rendering mode
+Fix feral PKer message
+kevlar jumpsuits use the correct requirement
+non integer additive values for enchants work
+AIM: Fix some instances of stack changing list positions while moving items
+Improvement & renovation to house_01 abandoned version
+Stop dodging good spells, fix uncanny dodge spell crash
+Items that can only be repaired with Budget Steel can be repaired
```
Comment (dseguin) [inline data/changelog.txt:275]:
```suggestion
Prevent car from spawning into a house wall
HP widgets gets equivalent bodypart
Fix grainy glyphs in blended font rendering mode
Stop dodging good spells, fix uncanny dodge spell crash
```

File: data/changelog.txt [hunk 6]
```diff
@@ -229,11 +296,14 @@ Refactor timer items to be a bit more time based
...
 Carrier for items on ground is nullptr
 Tick_action as a separate thing from use_action
+New documentation on how to test proposed changes
 
 
 ## Build:
 Support Mac arm64 build
 Adds VS Code Dev Containers & Workspace Config
+Add more '#pragma once' guards
+Add documentation for using devcontainers in VSCode
```
Comment (dseguin) [inline data/changelog.txt:306]:
```suggestion
```

File: data/changelog.txt (hunk 2 above)
Comment (Maleclypse) [reply data/changelog.txt:95]: I think the mutable stream is Innawoods only?

Comment (dseguin) [reply data/changelog.txt:92]: No problem, in that case I think the line should be moved over to the mods section

File: data/changelog.txt [hunk 7]
```diff
@@ -72,6 +73,7 @@ Adds new skateboard vehicle
 Most scenarios now provide vision of the nearest city on start
 Alternative resolution to clean back bay quest
 Add a synagogue
+Adds a mutable stream to the mapgen
```
Comment (Maleclypse) [inline data/changelog.txt:76]:
```suggestion
```

File: data/changelog.txt [hunk 8]
```diff
@@ -147,6 +150,8 @@ Lycra is less protective
 BMI has a less all consuming impact on how healthy you are
 Being badly wounded will always allow you to swap characters in camp while you heal
 Intelligence provides a multiplier to current focus rather than adding to its value
+More monsters fight back if cornered
+Higher dodge skill lowers the stamina cost to dodge
```
Comment (Maleclypse) [inline data/changelog.txt:154]:
```suggestion
Higher dodge skill lowers the stamina cost to dodge
Adds a mutable stream to the mapgen
```

**Pull request merged**